sources/web/datalab/polymer/modules/ApiManager.ts

```diff
@@ -60,6 +60,7 @@ interface JupyterFile {
 interface UserSettings {
```

Comment:
Looks like I forgot to remove this, this is defined and maintained in `common.d.ts` now along with `AppSettings`. I think we want to add the new field there instead, and remove this interface definition altogether. :)